### PR TITLE
add underscore and update version

### PR DESCRIPTION
**Pull request by Jonahss**
will need to update package.json file in appium and other libs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "lib": "lib"
   },
   "dependencies": {
-    "q": "^1.1.2"
+    "q": "^1.1.2",
+    "underscore": "^1.8.2"
```
Comment (jlipps) [inline package.json]: can we use lodash instead? i've been trying to move everything over to it

**Comment by jlipps** [comment]
also, this only needs to be one commit, really.

**Comment by Jonahss** [comment]
lodash now.
squashed to one commit.

**Comment by jlipps** [comment]
ok, will you take care of publishing and updating downstream package.jsons?

**Comment by Jonahss** [comment]
yup.

**Pull request merged**